ui/job-view/pushes/JobsAndGroups.jsx

```diff
@@ -18,6 +18,27 @@ export default class JobsAndGroups extends React.Component {
       toggleSelectedRunnableJob,
     } = this.props;
 
+    const confirmGroups = {};
+    for (const g of groups) {
+      // group.mapKey == pushID groupSymbol Tier platform buildtype
+      // find matching group.mapKey
+      if (g.symbol.endsWith('-cf')) {
+        let foundTier = false;
+        let gname = '';
+        for (let tier = 1; tier <= 3; tier++) {
+          gname = g.mapKey.replace('-cf3', tier);
```

Comment:
Could you explain where and how the `-cf3` gets created?

Reply:
as seen here: 
https://github.com/mozilla/treeherder/blob/1b0aa257f0d599e274d86dc3db6fa26d36bd26c0/ui/helpers/aggregateId.js#L17

this ends up squashing many items together to make a "unique" mapkey.